satpy/readers/iasi_l2.py

```diff
@@ -175,3 +175,33 @@ def _form_datetimes(days, msecs):
         all_datetimes.append(scanline_datetimes)
 
     return np.array(all_datetimes, dtype=np.float64)
+
+
+class IASIL2CDRNC(NetCDF4FsspecFileHandler):
+    """Reader for IASI L2 CDR in NetCDF format.
+
+    Reader for IASI All Sky Temperature and Humidity Profiles - Climate
+    Data Record Release 1.1 - Metop-A and -B. Data and documentation are
+    available from http://doi.org/10.15770/EUM_SEC_CLM_0063. Data are
+    also available from the EUMETSAT Data Store under ID EO:EUM:DAT:0576.
+    """
+
+    def get_dataset(self, data_id, ds_info):
+        """Obtain dataset."""
+        ds = self[data_id["name"]]
+        if "scan_lines" in ds.dims:
+            ds = ds.rename(scan_lines="y")
+        if "pixels" in ds.dims:
+            ds = ds.rename(pixels="x")
+        return ds
```

Comment:
Does any masking/_FillValue handling have to happen here? Or is the default scale factor/add offset stuff work well enough?

Reply:
I don't know.  What might I have to do here?

Reply:
I have added `_FillValue` masking to 01b5219e9.

Not sure what to do about `CloudFraction`, which is type uint8 and would explode to float64 if setting values to nan.  I've settled to not masking this variable at all.